Apps/W1/HybridBaseDeployment/app/src/pages/IntelligentCloudStatFactbox.Page.al

```diff
@@ -180,6 +181,37 @@ page 4008 "Intelligent Cloud Stat Factbox"
                             Page.Run(Page::"Hybrid Companies List");
                         end;
                     }
+
+                    field("Validation Errors"; ValidationErrors)
```

Comment:
As we have talked we should get a status per test (validator) that can be drilled into specific validators.

Reply:
This opens a list page that has drill down functionality per test.